unity-renderer/Assets/Scripts/MainScripts/DCL/Controllers/HUD/PlayerInfoCardHUD/PlayerInfoCardHUDController.cs

```diff
@@ -152,13 +160,34 @@ private void SetUserProfile(UserProfile userProfile)
         view.SetIsBlocked(IsBlocked(userProfile.userId));
         LoadAndShowWearables(userProfile);
         UpdateFriendshipInteraction();
+        view.HideActivationDate();
+
+        GetProfileFromCatalyst(userProfile)
+            .Then(catalystProfile =>
+            {
+                if (catalystProfile == null) return;
+                view.ShowActivationDate(catalystProfile.GetActivationDate());
+            });
```

Comment:
could you assign the fetch to a field to be able to (and) dispose it later? (like in `CloseCard` and `Dispose` methods)
like:
`Promise<string> profilePromise = GetProfileFromCatalyst(userProfile).Then(...)`
`profilePromise.Dispose()`
since pending promises could lead to unwanted errors during tests run?